ui-core/src/components/inputs/StatusMessage.tsx

```diff
@@ -1,30 +1,50 @@
 import React from 'react';
 
+import { X } from '@osrd-project/ui-icons';
+
 import cx from 'classnames';
 
 import InputStatusIcon from './InputStatusIcon';
 
 export type Status = 'success' | 'info' | 'error' | 'warning' | 'loading';
+export type TooltipType = 'left' | 'right';
 
 export type StatusWithMessage = {
+  tooltip?: TooltipType;
   status: Status;
   message?: string;
 };
 
 export type StatusMessageProps = {
   statusWithMessage: StatusWithMessage;
   showIcon?: boolean;
+  showClose?: boolean;
   small?: boolean;
 };
 
-const StatusMessage = ({ statusWithMessage, showIcon, small }: StatusMessageProps) => {
-  const { status, message } = statusWithMessage;
+const StatusMessage = ({ statusWithMessage, showIcon, showClose, small }: StatusMessageProps) => {
+  const { tooltip, status, message } = statusWithMessage;
+  const [close, setClose] = React.useState(false);
   if (message === undefined) return null;
 
   return (
-    <div className="status-message-wrapper">
+    <div
+      className={cx({
+        'status-message-wrapper': !tooltip,
+        'status-message-wrapper-tooltip': tooltip,
+        'tooltip-left': tooltip === 'left',
+        'tooltip-right': tooltip === 'right',
+        'tooltip-close': showClose && close,
+        [status]: status,
+      })}
+    >
       {showIcon && <InputStatusIcon status={status} small={small} />}
       <span className={cx('status-message', { [status]: status })}>{message}</span>
+      {showClose && (
```

Comment:
As we already have access to `statusWithMessage` in this component, I don't think we need the `showClose` prop, we could just do

```suggestion
      {statusWithMessage.status === 'info' && (
```